layouts/partials/footer.html

```diff
@@ -49,27 +55,30 @@
 <script type="text/javascript">
   go();
   window.addEventListener('resize', go);
-  function go(){
-    var d=document.querySelector(".desktop")
-    if(document.documentElement.clientWidth > 768) {
-      d.style.display="";
+  function go() {
+    var d = document.querySelector(".desktop")
+    if (document.documentElement.clientWidth > 768) {
+      d.style.display = "";
     } else {
-      d.style.display="none";
+      d.style.display = "none";
     }
   }
 </script>
 <script type="text/javascript">
-  var a=document.querySelector("#toggle-btn"),b=document.querySelector("#toggle-content");a.addEventListener("click",function(){""==b.style.display?(b.style.display="none",a.innerHTML="☰"):(b.style.display="",a.innerHTML="X")});</script>
+  var a = document.querySelector("#toggle-btn"), b = document.querySelector("#toggle-content"); a.addEventListener("click", function () { "" == b.style.display ? (b.style.display = "none", a.innerHTML = "☰") : (b.style.display = "", a.innerHTML = "X") });</script>
```

Comment:
This is minified to reduce space! Hence why there are no spaces between the expressions.